scripts/validateContractDeployment.ts

```diff
@@ -66,40 +66,6 @@ interface ContractInfo {
   signalledRoles: string[];
 }
 
-const expectedRoles = {
-  CONFIG_KEEPER: ["ConfigSyncer"],
-  ROLE_ADMIN: ["Timelock"],
-  ROUTER_PLUGIN: [
-    "ExchangeRouter",
-    "SubaccountRouter",
-    "GlvRouter",
-    "GelatoRelayRouter",
-    "SubaccountGelatoRelayRouter",
-  ],
-  CONTROLLER: [
-    "OracleStore",
-    "MarketFactory",
-    "GlvFactory",
-    "Config",
-    "ConfigSyncer",
-    "Timelock",
-    "Oracle",
-    "SwapHandler",
-    "AdlHandler",
-    "DepositHandler",
-    "WithdrawalHandler",
-    "OrderHandler",
-    "ExchangeRouter",
-    "LiquidationHandler",
-    "SubaccountRouter",
-    "ShiftHandler",
-    "GlvHandler",
-    "GlvRouter",
-    "GelatoRelayRouter",
-    "SubaccountGelatoRelayRouter",
-  ],
-};
-
```

Comment:
@Call-Sol jfyi i've moved expectedRoles to config/roles because i also need them in validateRolesUtils